deepsea.spec

```diff
@@ -32,35 +32,22 @@ Url:            https://github.com/suse/deepsea
 Source0:        %{name}-%{version}.tar.bz2
 
 BuildRequires:  salt-master
+BuildRequires:  python-setuptools
 Requires:       salt-master
 Requires:       salt-minion
 Requires:       salt-api
 Requires:       python-ipaddress
 Requires:       python-netaddr
 Requires:       python-rados
+Requires:       python-setuptools
```

Comment:
why is `python-setuptools` both a build dependency and a runtime dependency?

Reply:
It's necessary for the "console script" (deepsea command)  created by the setuptools installer to work.